layouts/_partials/head.html

```diff
@@ -21,9 +21,9 @@
     src="{{ $jsAppearance.RelPermalink }}"
     integrity="{{ $jsAppearance.Data.Integrity }}"
   ></script>
-  {{ $cssScheme := resources.Get (printf "css/schemes/%s.css" (lower .Site.Params.colorScheme | default "congo")) }}
+  {{ $cssScheme := resources.Get (printf "css/schemes/%s.css" (lower .Site.Params.colorScheme | default "cherry")) }}
```

Comment:
Kept the ability to apply a new scheme, it might be nice to change colors every now and then, but default to cherry (which we use) since we likely won't need the rest